gcovr/html_generator.py

```diff
@@ -72,17 +84,87 @@ def calculate_coverage(covered, total, nan_value=0.0):
     return nan_value if total == 0 else round(100.0 * covered / total, 1)
 
 
-def coverage_to_color(coverage, medium_threshold, high_threshold):
+def coverage_to_class(coverage, medium_threshold, high_threshold):
     if coverage is None:
-        return 'LightGray'
-    elif coverage < medium_threshold:
-        return low_color
-    elif coverage < high_threshold:
-        return medium_color
-    else:
-        return high_color
-
-
+        return 'coverage-unknown'
+    if coverage == 0:
+        return 'coverage-none'
+    if coverage < medium_threshold:
+        return 'coverage-low'
+    if coverage < high_threshold:
+        return 'coverage-medium'
+    return 'coverage-high'
+
+
+class RootInfo(object):
+
+    def __init__(self, options):
+        self.options = options
```

Comment:
Would it be possible to extract the required options here, instead of storing them all? I think that would make the data flow clearer (see also the comment about the simple1-stdout test).

Reply:
done